rebar.config

```diff
@@ -20,12 +22,12 @@
        ]}.
 
 {relx, [{release, {aerepl, "2.0.0"},
-         [aerepl, {aechannel, load}, {aecore, load}, {aeutils, load}]},
-        {lib_dirs, ["node/_build/dev1/rel/aeternity/lib"]},
+         [aerepl, {aechannel, load}, {aecore, load}, {aeutils, load}, {mnesia, load}, sasl]},
+        {lib_dirs, ["node/_build/dev1/rel/aeternity/lib", "_build/prod/lib/"]},
```

Comment:
Why do you need to load`mnesia` and include `_build/prod/lib`?

Reply:
There were some errors in the release. Didn't affect semantics as far as I checked, but I prefer clean run just in case